src/main/kotlin/teaforge/platform/RoboRio/internal/Subscriptions.kt

```diff
@@ -14,11 +15,8 @@ sealed interface SubscriptionState<TMessage> {
         data class DioPortValue<TMessage>(
                 val config: Subscription.DioPortValue<TMessage>,
                 val lastReadTimeMicroseconds: Long,
-        ) : SubscriptionState<TMessage>
-
-        data class DioPortValueChanged<TMessage>(
-                val config: Subscription.DioPortValueChanged<TMessage>,
-                val lastReadValue: DioPortStatus,
+                val active: Boolean,
+                val hasInit: Boolean
```

Comment:
These values seem to be communicating initialization and error states. Is this to handle the case when you subscribe to a DioPort that is configured for output?